lib/model/rwfolder.go

```diff
@@ -288,13 +302,20 @@ func (f *sendReceiveFolder) Serve() {
 			default:
 				l.Infoln("Completed initial scan (rw) of", f.Description())
 				close(f.initialScanCompleted)
+				if fsWatcher.WatchingFs {
+					f.delayFullScan()
```

Comment:
This only runs after the first scan - on the next scan it'll never be rescheduled, so we only ever get two full scans. We need to reschedule a full scan one long interval after each full scan, unless I misunderstand.